src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -1616,7 +1619,7 @@ static void Main(string[] args)
 
     private static int NewMethod(int v, int i)
     {
-        {|Warning:v = v + i;|}
```

Comment:
why did this warning go away?

Reply:
I think it has to do with the warning message that was deleted from [this](https://github.com/dotnet/roslyn/pull/39745#discussion_r349804976) discussion.